### PR TITLE
Clean up some golint errors unrelated to comments in kubeapiserver

### PR DESCRIPTION
**Pull request by wgliang**
**What this PR does / why we need it**:

Clean up some golint errors unrelated to comments in kubeapiserver.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

/kind cleanup
/area apiserver

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubeapiserver/admission/initializer_test.go [hunk 1]
```diff
@@ -33,8 +33,8 @@ type WantsCloudConfigAdmissionPlugin struct {
 	cloudConfig []byte
 }
 
-func (self *WantsCloudConfigAdmissionPlugin) SetCloudConfig(cloudConfig []byte) {
-	self.cloudConfig = cloudConfig
+func (p *WantsCloudConfigAdmissionPlugin) SetCloudConfig(cloudConfig []byte) {
```
Comment (yue9944882) [inline pkg/kubeapiserver/admission/initializer_test.go]: is this required by golint?

**Review by yue9944882 — CHANGES_REQUESTED** [review]
thx for fixing this @wgliang 

plz keep a minimal change w/ golint

File: pkg/kubeapiserver/authenticator/config.go [hunk 2]
```diff
@@ -45,7 +45,7 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 )
 
-type AuthenticatorConfig struct {
+type Config struct {
```
[the remark this answers is not in the document]
Comment (yliaog) [reply pkg/kubeapiserver/authenticator/config.go]: +1

[the remark this answers is not in the document]
Comment (wgliang) [reply pkg/kubeapiserver/authenticator/config.go]: OK, the issues reported by golint are mainly related to AuthenticatorConfig. Such as:

> type name will be used as authenticator.AuthenticatorConfig by other packages, and that stutters; consider calling this Config


If this is not necessary, I will close this PR.

**Pull request closed without merging**